components/Card/index.tsx

```diff
@@ -0,0 +1,21 @@
+import Image from 'next/image'
+import { PersonRecord } from 'types'
+
+const Card = ({ name, title, department, avatar }: PersonRecord) => {
+	return (
+		<div data-department={department.id}>
+			<Image
+				src={avatar?.url ? avatar.url : '/assets/default.png'}
```

Comment:
```suggestion
				src={avatar?.url ?? '/assets/default.png'}
```

You can simplify these with the nullish coalescing operator!